docs/aks-installation.md

```diff
@@ -73,7 +73,7 @@ you can deploy Antrea as follows:
     ```
 
     To deploy the latest version of Antrea (built from the main branch), use the
-checked-in [deployment yaml](/build/yamls/antrea-aks.yml):
+checked-in [deployment yaml](https://github.com/antrea-io/antrea/blob/main/build/yamls/antrea-aks.yml):
```

Comment:
I believe @tnqn mentioned that the root cause is we should use relatively path in the issue comment.
you should update it as following one.
```suggestion
checked-in [deployment yaml](./build/yamls/antrea-aks.yml):
```

Reply:
This wouldn't work because it expands to wrong path and searches for the file in antrea website repo.
Actually any relative path wouldn't work here because these files (mostly yaml) are not present in antrea website github repo.

Reply:
If relative path has to work , the folder "build/yamls/"  needs to be created on https://antrea.io and then each file needs to be available there, otherwise will have to use absolute GitHub path

Reply:
I doubt this is the right way to fix the issue. If you change it to a link, then when you have any changes on a local file, it will jump to a website link which point to a main branch file. I don't think this is what we expected.
Let's see what's @antoninbas and @tnqn 's comments.

Reply:
All the docs should use relative paths. The link should be `../build/yamls/antrea-aks.yml` here

The right way to fix it is in the scripts which generate the website source (they should do link substitution when applicable: `../build/yamls/antrea-aks.yml -> https://github.com/antrea-io/antrea/blob/<CORRECT TAG>/build/yamls/antrea-aks.yml`)